DataFormats/FEDRawData/interface/FEDNumbering.h

```diff
@@ -108,6 +110,10 @@ class FEDNumbering {
      MAXSiPixeluTCAFEDID = 1349,
      MINTriggerUpgradeFEDID = 1350,
      MAXTriggerUpgradeFEDID = 1409,
+     MINCTPPSPixelsFEDID = 1462,
+     MAXCTPPSPixelsFEDID = 1466,
+     MINGEMFEDID = 1467,
+     MAXGEMFEDID = 1472,
```

Comment:
Indeed! This is the consequence of the cherry-pick of 8b278dd0abe9ac616f18588bee744d6c6649a584 (since you were interested in keeping in sync the FEDs range for the diamonds, and @mommsen wanted the documentation to be backported too)